packages/yoroi-extension/app/styles/overrides/revamp/Button.js

```diff
@@ -146,7 +150,7 @@ function getOutlinedStyles(variant: 'primary' | 'secondary', theme: Object): Obj
 
 function getTertiaryStyles(variant: 'primary' | 'grayscale', theme: Object): Object {
   return {
-    backgroundColor: 'transparent',
+    backgroundColor: theme.palette[variant][50],
```

Comment:
On the button tertiary styles, we see the bg is transparent. I think this is a mistake. The default state is without bg and if you hover or press or any other action, there's a specific bg style to be applied. This works for both _Text button_ (previously Tertiary) variants. Check out on figma: https://www.figma.com/file/jfSkOH2pMk8stPw7rCjn4C/Components---Web?type=design&node-id=1777%3A26125&mode=dev